clang/include/clang/Basic/Attr.td

```diff
@@ -1197,6 +1197,13 @@ def SYCLDevice : InheritableAttr {
   let Documentation = [SYCLDeviceDocs];
 }
 
+def SYCLGlobalVar : InheritableAttr {
+  let Spellings = [GNU<"sycl_global_var">];
```

Comment:
Why does this only have a GNU spelling?

Reply:
Because the SYCLDevice attribute above it also has only GNU spelling and I mimicked surrounding code. I'd imagine any spelling or combination of spellings would be fine. Is there a preference? Fewer spellings means less surface area and fewer tests, especially for an attribute that folks are already planning to use only via a macro.

Reply:
We're sadly inconsistent here -- `sycl_device` gets only a GNU spelling while `sycl_kernel` gets the Clang spelling. :-/ I don't have a strong preference, so I'd say we're fine to go with the GNU spelling for now. We can add additional spellings later if we need to.